static/task-management-template/index.html

```diff
@@ -0,0 +1,202 @@
+          <p>convert more of your website visitors into loyal customers with personalized it</p>
+        </article>
+      </div>
+    </div>
+
+    <!-- track user part -->
+
+    <div class="sm:flex sm:justify-center pt-20">
+      <div class="px-5 py-15 sm:w-1/2 m-20">
+        <h3 class="text-3xl font-semibold pt-10">
+          Track your users
+        </h3>
+        <p>
+          There are many variations of passages of and lorem ipsum available, but the majority files
+        </p>
+        <div class="flex item-center">
+          <img src="./assets/rename-img.svg" alt="">
+          <div class="px-1 py-3">
+            <h4>See how it works.</h4>
+            <a href="#" class="text-blue-500 hover:text-blue-700">Watch demo -></a>
+          </div>
+        </div>
+      </div>
+      <img src=".\assets\analytics-img.svg" class="max-w-full h-auto sm:w-1/2" alt="..." />
+    </div>
+
+    <!-- track audience part -->
+
+    <div class="sm:flex sm:justify-center sm:flex-row-reverse pt-10">
+      <div class="px-5 py-5 sm:w-1/2 m-20">
+        <h3 class="text-3xl font-semibold pt-10">
+          Track audience
+        </h3>
+        <p>
+          There are many variations of passages of and lorem ipsum available, but the majority files
+        </p>
+        <div class="flex item-center">
+          <img src="./assets/rename-img.svg" alt="">
+          <div class="px-1 py-3">
+            <h4>See how it works.</h4>
+            <a href="#" class="text-blue-500 hover:text-blue-700">Watch demo -></a>
+          </div>
+        </div>
+      </div>
+      <div class="sm:w-1/2"><img src=".\assets\aud-img.svg" class="max-w-full h-auto" alt="..." /></div>
+    </div>
+
+    <!-- learn more and ending part -->
+
+    <div class="text-center px-20 py-10">
+      <div class="xl:px-100 m-10">
+        <p><h3 class="text-3xl font-semibold py-4">Learn more about our Conversational support</h3></p>
+        <button class="bg-blue-500 hover:bg-blue-700 text-white font-semibold px-4 py-2 rounded">Request a demo</button>
+      </div>
+    </div>
+
+    <footer class="px-5 sm:px-10 lg:px-20">
+    <div class="sm:flex">
+      <div class="flex flex-col items-center px-6 py-4 sm:items-start sm:w-5/12">
+        <img src="./assets/Logo.svg" alt="">
+        <p class="text-center sm:text-left">We help business teams increase operational efficiency, not time spent building admin tools.</p>
+
+        <div class="py-4 flex flex-row">
+          <a href="#"><img src="./assets/facebook.svg" alt="" class="px-4"></a>
+          <a href="#"><img src="./assets/twitter.svg" alt="" class="px-4"></a>
+          <a href="#"><img src="./assets/LinkedIn.svg" alt="" class="px-4"></a>
+        </div>
+      </div>
+
+            
+      <div class="flex justify-between px-6 py-6 sm:w-7/12">
+
+        <div class="w-1/3">
+          <h4 class="font-semibold text-lg">Product</h4>
+          <p><a href="#" class="text-gray-400 hover:text-gray-600">Features</a></p>
```

Comment:
For links, semantically correct would be to use <ul> & <li> elements instead of <p>